QUICKSTART.md

```diff
@@ -0,0 +1,226 @@
+# Quick Start Guide
```

Comment:
BTW, troubleshooting from this file has been moved to https://github.com/TrenchBoot/TrenchBoot.github.io/blob/cleanup/docs/user-docs/troubleshooting.md#failed-start-with-intel-txt, please check if it needs updating and we can link to that here instead of repeating the text.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/33/commits/ce3e56e8094ac4c32181ea20680a7f48dd3a291f